src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -357,10 +357,13 @@ while ( $counter <= ${NUM_SGMT} )
    
    if( -e EGRESS.ldas ) then
       set rc = 0
+      echo GEOSldas Run Status: $rc
    else
       set rc = -1
+      echo GEOSldas Run Status: $rc
+      echo "GEOSldas run failed, exit without post-processing"
+      exit
```

Comment:
Do we really want to exit here?  If we do, there will not  be an attempt at post-processing.  I would remove the "exit" here and in the preceding line say something like: 
`echo "GEOSldas run FAILED, attempting post-processing..."`
Then also move the "if ... exit" statement to occur ~line 714 (see comment below).

Reply:
This one should be "break" which comes out of the loop